include/rxmesh/reduce_handle.h

```diff
@@ -119,8 +119,37 @@ class ReduceHandle
         return std::sqrt(reduce_2nd_stage(stream, cub::Sum(), 0));
     }
 
+    //does either arg max or min depending on second argument is_min
+    T arg_maxOrmin(const Attribute<T, HandleT>& attr,
```

Comment:
I would rather have two functions one for `arg_max` and another for `arg_min` to avoid this overloaded naming